### PR TITLE
sdrangel: new port

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description

SDRangel is an Open Source Qt5 / OpenGL 3.0+ SDR and signal analyzer frontend to various hardware.

Also include serialDV as dependency.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.15 19A602
Xcode 11.1 11A1027

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/SDRangel/Portfile [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+variant native description {Enable native cpu flags (recommended)} {
+    configure.args-replace  -DFORCE_SSE41=ON -DFORCE_SSE41=OFF
+}
+
+variant debug description {Enable debug messages} {
+    configure.args-append   \
+        -DDEBUG_OUTPUT=ON   \
+        -DCMAKE_BUILD_TYPE=Debug
+}
```
Comment (mf2k) [inline science/SDRangel/Portfile]: Please sort the variants alphabetically.

File: science/SDRangel/Portfile [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+long_description    ${description}
+
+github.setup        f4exb sdrangel 4.11.11 v
+checksums           rmd160  5364bb8f90fb7d4c47e6f946ab59f67618451d1c \
+                    sha256  20046e245a7199b1254bea064a3e1da95bd2ef11c68d176b15f7836fd57b3544 \
+                    size    24138069
+revision            0
+
+compiler.c_standard 2011
+compiler.cxx_standard 2011
```
Comment (mf2k) [inline science/SDRangel/Portfile]: Minor nitpick: This does not follow the mode line spacing.

File: science/serialDV/Portfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    based devices in packet mode over a serial link.
+long_description    ${description}
+
+github.setup        f4exb serialDV 1.1.4 v
+checksums           rmd160  30fcb2b18a28565772435b5e4c048dd13cee4a88 \
+                    sha256  b13851a1ef743cf683f80ecb788cc3050fdc01307e52b0656a2d788d92bf5ee1 \
+                    size    297844
+revision            0
+
+compiler.cxx_standard 2011
```
Comment (mf2k) [inline science/serialDV/Portfile]: Minor nitpick: This does not follow the mode line spacing.

**Comment by michaelld** [comment]
Cool. Let me give this a shake-down!

File: science/SDRangel/Portfile (hunk 1 above)
Comment (ra1nb0w) [reply science/SDRangel/Portfile]: thank you!

**Comment by michaelld** [comment]
Waiting for CI ...

**Comment by ra1nb0w** [comment]
Travis: Fetching distfile failed

So everything is good.

File: science/SDRangel/Portfile (hunk 1 above)
Comment (mf2k) [reply science/SDRangel/Portfile]: I'm not seeing that here.

Comment (ra1nb0w) [reply science/SDRangel/Portfile]: ops push.
variants are now organized in alphabetically order per type.
I mean, first the generic one than driver one to easily manage future update.

**Comment by ra1nb0w** [comment]
thank you!

**Pull request merged**